### PR TITLE
Player 3986 - Update Node to 8.11.3

### PR DESCRIPTION
**Pull request by aeng7**
Updating Node to 8.11.3 and allow for usage of ES6. DO NOT MERGE YET! Opening just for review. We'll need to update html5-common and the build scripts to accommodate for these Node updates

The html5-common submodule is currently synced to the player-3896 branch in the html5-common repo. There is a package.json update needed in the html5-common repo that would break things unless we're using node 8.11.3. When all the Node 8.11.3 PRs are ready and approved, I'll create a PR in the html5-common repo to push the package.json change and update these other PRs with the submodule update.

### CODE REVIEW

File: tests/setup.js [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+OO.$ = global.window.$;
+
+global.expect = require('expect.js');
+
+// a wrapper domparser simulating Mozilla DOMParser in the browser:
+window.DOMParser = function() {};
+
+require.requireActual("../html5-common/js/utils/InitModules/InitOOUnderscore.js");
+
+OO._.extend(window.DOMParser.prototype, {
```
Comment (jpfranco) [inline tests/setup.js]: Would it be better to use `Object.assign`, `Array.forEach`, etc., in order to minimize dependency on `OO` and underscore?

File: tests/unit/main_html5/wrapper-api-tests.js [hunk 2]
```diff
@@ -88,27 +92,28 @@ describe('main_html5 wrapper tests', function () {
...
-      wrapper.setVideoUrl("url", OO.VIDEO.ENCODING.HLS, true);
-    }
-
-    element.__dvrSpy = {
-      restore: function() {
-        element.seekable.start = startSpy.originalValue;
-        element.seekable.end = endSpy.originalValue;
-      }
-    };
+    //if (!element.__dvrSpy) {
```
Comment (jpfranco) [inline tests/unit/main_html5/wrapper-api-tests.js]: Do we need to keep this commented code?

**Comment by jpfranco** [comment]
On a general note, can we add a description in the PR title? This helps identify each individual PR when viewing a list of PR's

File: tests/setup.js (hunk 1 above)
Comment (aeng7) [reply tests/setup.js]: Yep, will change. I copied this from test_lib.js and didn't notice. Nice catch!

File: package.json [hunk 3]
```diff
@@ -10,23 +10,28 @@
     "test": "jest"
   },
   "devDependencies": {
-    "browserify": "11.0.1",
-    "expect.js": "0.3.0",
+    "browserify": "16.2.2",
```
Comment (jpfranco) [inline package.json]: Might also be worth adding the `engines` property

Comment (aeng7) [reply package.json]: Added

**Comment by pilievOoyala** [comment]
LGTM

**Pull request merged**